src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_custom.py

```diff
@@ -820,6 +821,43 @@ def test_aks_stop(self):
         )
         self.assertEqual(aks_stop(self.cmd, self.client, "rg", "name", False), None)
 
+    @mock.patch('azure.cli.command_modules.acs.custom.prompt_y_n', return_value=True)
```

Comment:
Please add a live test (e2e test) to `test_aks_commands.py`

Reply:
live test added